### PR TITLE
Update setup.sh Added missing role and reuse of previous service account

### PR DESCRIPTION
**Pull request by cvalla**
Added -s option in order to update a previous service account 
Added a new role Service Account Key Admin requested during terraform execution

### CODE REVIEW

**Comment by cvalla** [comment]
Hi @kevensen,
I'm deploying forseti on a brand new project on GKE using gke_end_to_end.
I'm using the cloud shell to do so.
I encountered problems on 4.3.0 and so moved to 4.1.0 (an error regarding ssh_key output).
During Terraform execution I viewed an error regarding lack of permissions .
Below the terraform code used

module "forseti-on-gke-end-to-end" {
source = "terraform-google-modules/forseti/google//examples/on_gke_end_to_end"
version = "4.1.0"
credentials_path = "credentials.json"
domain = "xxxxx"
gsuite_admin_email = "admin@xxxxx"
org_id = "1234567890"
project_id = "my-project"
region = "europe-west1"
zones = ["europe-west1-b"]
network_description = "Forseti GKE Network"
gke_node_ip_range = "10.1.0.0/20"
}

The updates I made to setup.sh are useful to CI/CD and corrects lack of permissions that cause an error

**Comment by kevensen** [comment]
@cvalla Thanks for the additional information.  In v4.3.0 of the module, the GKE related roles did not yet exist in the _helper/setup.sh_ script.  At that time, we were asking users to manually enable the roles necessary for on-GKE outside the script, including "Service Account Key Admin".

In v5.0.0 of the module, we made the decision to use [GKE Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) removing the need for the role "Service Account Key Admin".

If you have not already, I recommend looking at v5.0.0 of this module as it does not need this role.

Please let me know if you have further questions.

File: helpers/setup.sh [hunk 1]
```diff
@@ -41,9 +36,11 @@ ORG_ID=""
 WITH_ENFORCER=""
 HOST_PROJECT_ID=""
 ON_GKE=""
+SERVICE_ACCOUNT_NAME="cloud-foundation-forseti-${RANDOM}"
+IS_UPDATE=0
 
 OPTIND=1
-while getopts ":hekf:p:o:" opt; do
+while getopts ":hekfs:p:o:" opt; do
```
Comment (kevensen) [inline helpers/setup.sh]: I think this needs to be `:hekf:s:p:o:`

File: helpers/setup.sh [hunk 2]
```diff
@@ -173,6 +201,11 @@ gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
     --role="roles/cloudsql.admin" \
     --user-output-enabled false
 
+gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
+    --member="serviceAccount:${SERVICE_ACCOUNT_EMAIL}" \
+    --role="roles/iam.serviceAccountKeyAdmin"
+    --user-output-enabled false
+
```
Comment (kevensen) [inline helpers/setup.sh]: Please remove as this role is not necessary in terraform-google-forseti v5.0.0.

**Comment by kevensen** [comment]
@cvalla Thanks for the updates.  I expect the integration tests will pass.  Once they do, I can merge.

**Pull request merged**